ci/centos-7/Dockerfile

```diff
@@ -1,5 +1,11 @@
 FROM centos:7
 
+ENV FLEX_VERSION=2.6.4 \
+    FLEX_DIR=/opt/flex
```

Comment:
Consistency with the other files says both of the lines should start with `ENV`.

Nitpicky, I know.